data/crac-file/crac-file-xlsx-api/src/test/java/com/farao_community/farao/data/crac_file/xlsx/service/ImportServiceTest.java

```diff
@@ -156,4 +156,23 @@ public void shouldImportXlsxCracFileFromFile20170215XlsxCracFrV09V23() throws Ex
 
         assertEquals("Topology RA 1", remedialAction.getName());
     }
+
+    @Test
+    public void shouldImportXlsxCracFileWithPstRemedialAction() throws Exception {
```

Comment:
I think it is better to add the modification of the pst name in the "test_crac_simple.xlsx" instead of creating another file for the same test case. You can so delete this method and add assertEquals("Pst element") to the method below shouldImportCorrectIdFromXlsxCracFile()